### DISCUSSION TITLE
Can you use Checks API Actions to trigger new workflows.

### DISCUSSION THREAD

**Discussion by lannonbr** [community/community#25826] [Actions]
<p>So I have a repo that I am trying to test out a sample project where 2 workflows exist.</p>
<p>Repo: <a href="https://github.com/lannonbr/linting-checks-workflow" rel="nofollow noopener">https://github.com/lannonbr/linting-checks-workflow</a></p>
<p>The first workflow, lint.yml runs on push and runs prettier. If prettier fails, it creates a new checkrun with an action. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/a2fb95976933726f1036077ad79da0e706bdef26.png" data-download-href="/uploads/short-url/nfOmLqXQaPFDYUgJjRTLChuAuJE.png?dl=1" title="Screenshot_2019-09-24 lannonbr linting-checks-workflow.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3685876/181093637-229b1fc0-bcb1-4cd6-a8c0-cfab5b49cf24.png" alt="Screenshot_2019-09-24 lannonbr linting-checks-workflow.png" data-base62-sha1="nfOmLqXQaPFDYUgJjRTLChuAuJE" width="490" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/a/a2fb95976933726f1036077ad79da0e706bdef26_2_490x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/a/a2fb95976933726f1036077ad79da0e706bdef26_2_735x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/a/a2fb95976933726f1036077ad79da0e706bdef26_2_980x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/a2fb95976933726f1036077ad79da0e706bdef26_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_2019-09-24 lannonbr linting-checks-workflow.png</span><span class="informations">1278×1302 35.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The second workflow, run-prettier.yml, I want to run when you click on that “Fix linting” action from the first workflow.</p>
<p>Is there a way I can do this so a check_run action can trigger another workflow as the 2nd workflow is never executing at all.</p>
<p>Thanks!</p>

**Comment by francisfuzz** [comment, marked as answer]
<p>Hey <a class="mention" href="/lannonbr">@lannonbr</a> ! Great question. Events raised from the Actions app do not trigger workflows as mentioned here:</p>
<p><a href="https://help.github.com/en/github/automating-your-workflow-with-github-actions/events-that-trigger-workflows#about-workflow-events" rel="nofollow noopener">https://help.github.com/en/github/automating-your-workflow-with-github-actions/events-that-trigger-workflows#about-workflow-events</a></p>
<p>For example, if a check run is created by GitHub Actions and includes a <code>requested action</code>, pressing that <code>requested action</code> will trigger the <code>check_run.requested_action</code> webhook event but no new workflows will run if the repository contains a workflow configured to run when the <code>check_run</code> event occurs. </p>
<p>However, if a check run is created by a different GitHub App and it presents a <code>requested action</code>, it is possible to trigger a workflow configured to run when the <code>check_run</code> event occurs as soon as it’s interacted with.</p>
<p>I hope this helps!</p>
<p>Related thread: <a href="https://github.community/t/actions-are-not-triggering-when-the-pr-was-created-using-api/16315">https://github.community/t5/GitHub-Actions/Actions-are-not-triggering-when-the-PR-was-created-using-API/m-p/32379</a></p>
